/**:
  ros__parameters:
    host_ip: "127.0.1.1"
    controller_port: 8000
    tracker_port: 8001
    tracker_name: "tracker_1"
    controller_name: "controller_1"
    tracker_link: "vive_tracker_link"
    controller_link: "vive_controller_link"
    buttons_topic: "joy_inputs"
